zulip_bots/zulip_bots/bots/googletranslate/test_googletranslate.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python
+
+from __future__ import absolute_import
+from __future__ import print_function
+
+import json
+
+from unittest.mock import patch
+from requests.exceptions import HTTPError, ConnectionError
+
+from zulip_bots.test_lib import BotTestCase
+
+from zulip_bots.bots.googletranslate.googletranslate import TranslateError
```

Comment:
This is unused; you don't have a test for the `TranslateError` case?
Maybe add a general `Exception` case too, since you handle that in the bot?